src/gsy_e/gsy_e_core/export.py

```diff
@@ -301,6 +335,27 @@ def _export_area_clearing_rate(self, area, directory, file_suffix, is_first) ->
         except OSError:
             _log.exception("Could not export area market_clearing_rate")
 
+    def _export_future_offers_bid_trades_to_csv_files(
+            self, future_markets: "FutureMarkets", market_member: str, directory: dir,
+            filename: str, labels: Tuple, is_first: bool = False) -> None:
+        """ Export files containing individual future offers, bids (*-bids*/*-offers*.csv files).
+        """
+        file_path = self._file_path(directory, filename)
+        try:
+            with open(file_path, "a") as csv_file:
+                writer = csv.writer(csv_file)
+                if is_first:
+                    writer.writerow(labels)
+                if not future_markets.market_time_slots:
+                    return
+                time_slot = future_markets.market_time_slots[0]
```

Comment:
Why are you only selecting the first future market here? Is it because it already contains all the offers and bids and trades because it will be closed in the next market cycle?

I predict that researchers will like to look at the state of the other future markets too (e.g. when pausing the simulation). But lets wait until they ask for it. It will not be a big deal to add.

Reply:
Unfortunately it is a bit trickier, because I wanted to avoid duplicate entries. If I iterated through all market slots here, then all bids and offers from all future markets would be written to the csv files on every slot. This means that bids and offers that have been already recorded would be rewritten to the csv files. 
This can be avoided by filtering bids and offers through creation time, however I think it is cleaner to only write them to CSVs when the time slot is closing. If the researchers demand it then we can always enforce the workaround I mentioned, however I do not think they will, because I am not sure how frequently do they pause the simulation and look at the CSV files content (I would expect that they run the simulation first, analyse its results later).

Reply:
I see. OK. Definitely agree that filtering for creation time would be an overkill for now. Thanks!